polls/migrations/0006_experimenttype_experimenttypetaskpresentation_and_more.py

```diff
@@ -0,0 +1,100 @@
+# Generated by Django 4.1.1 on 2023-02-24 18:43
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("polls", "0005_alter_batch_created_at"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="ExperimentType",
+            fields=[
+                (
+                    "id",
+                    models.BigAutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                (
+                    "type",
+                    models.CharField(
+                        choices=[("2AFC", "2AFC"), ("A/B", "A/B")], max_length=10
+                    ),
+                ),
+            ],
+        ),
+        migrations.CreateModel(
+            name="ExperimentTypeTaskPresentation",
+            fields=[
+                (
+                    "id",
+                    models.BigAutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("task_presentation", models.CharField(max_length=50)),
+                (
+                    "experiment_type",
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.CASCADE,
+                        to="polls.experimenttype",
+                    ),
+                ),
+            ],
+        ),
+        migrations.CreateModel(
+            name="ExperimentTypeAnnotation",
+            fields=[
+                (
+                    "id",
+                    models.BigAutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("annotation", models.CharField(max_length=50)),
+                (
+                    "experiment_type",
+                    models.ForeignKey(
+                        on_delete=django.db.models.deletion.CASCADE,
+                        to="polls.experimenttype",
+                    ),
+                ),
+            ],
+        ),
+        migrations.CreateModel(
+            name="Experiment",
+            fields=[
+                (
+                    "id",
+                    models.BigAutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("name", models.CharField(max_length=50)),
```

Comment:
so short why

Reply:
It was just an initial push. I guess length of 100 will do.

Reply:
@najamF18 seems you forgot to extend this

Reply:
> @najamF18 seems you forgot to extend this

Thanks for reviewing it. I have updated it. Actually we removed these models and added them again. It may have skipped my mind.